### PR TITLE
Fix service bus mapping

### PR DESCRIPTION
**Pull request by trask**
Based on @lmolkova's https://github.com/Azure/azure-sdk-for-java/pull/21668#discussion_r685572619 and https://github.com/Azure/azure-sdk-for-java/pull/21668#pullrequestreview-725874372

### CODE REVIEW

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java [hunk 1]
```diff
@@ -129,20 +99,55 @@
...
+      AttributeKey.longKey("x-opt-enqueued-time");
 
   private static final OperationLogger exportingSpanLogger =
       new OperationLogger(Exporter.class, "Exporting span");
 
   private static final OperationLogger parsingHttpUrlLogger =
       new OperationLogger(Exporter.class, "Parsing http.url");
 
+  static {
+    Set<String> dbSystems = new HashSet<>();
```
Comment (heyams) [inline agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java:118]: does otel java sdk have dbSystems defined already? can we refer to that directly if it becomes available? i remember there is a common spec for dbsystems on the otel spec repo.

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java [hunk 2]
```diff
@@ -129,20 +99,55 @@
...
+    dbSystems.add("sqlite");
+    dbSystems.add("other_sql");
+    dbSystems.add("hsqldb");
+    dbSystems.add("h2");
+
+    SQL_DB_SYSTEMS = Collections.unmodifiableSet(dbSystems);
+
+    // TODO need to keep this list in sync as new semantic conventions are defined
+    // TODO make this opt-in for javaagent
+    Set<String> standardAttributesPrefix = new HashSet<>();
```
Comment (heyams) [inline agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java:135]: same here.. i feel that otel should take care of all the standdard attribute prefixes.

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java (hunk 1 above)
Comment (trask) [reply agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java:118]: 👍

File: agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java (hunk 2 above)
Comment (trask) [reply agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java:135]: unfortunately there are no otel constants we can use for this, at least not yet

**Review by heyams — APPROVED** [review]

**Pull request merged**